environment.yml

```diff
@@ -33,23 +37,24 @@ dependencies:
   - ioos-tools
   - ioos_qc
   - ipyleaflet
+  - jupyter
   - jupyter-rsession-proxy
+  - jupyter_contrib_nbextensions
+  - jupyterlab
   - matplotlib-base
   - movingpandas
   - nb_conda_kernels
   - nc-time-axis
   - netcdf4
   - numpy
-  - oceansdb
   - palettable
-  - pandas<1.1
+  - pandas
   - panel
   - param
   - podaacpy
-  - proj >=7
   - pyoos
```

Comment:
Is pyoos still maintained actively? It doesn't look like it.

Reply:
Does it need to be maintained to be useful? If so we need to remove half of the packages here :-)

I need it for an example BTW, that's why I'm keeping it there.

Reply:
>Does it need to be maintained to be useful? If so we need to remove half of the packages here :-)

Point taken :roll_eyes:. But you're using in an example, then it's needed.